### ISSUE TITLE
Tmux not receiving some keyboard input

### ISSUE THREAD

**Issue by TickleThePanda** [zeit/hyper#2721]
- [ ] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Microsoft Windows 10 Enterprise, Version	10.0.15063 Build 15063
- **Hyper.app version**: 2.0.0-canary.14
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: [.hyper.js](https://gist.github.com/TickleThePanda/94623390c432000d7aa6f5478d0a07dd)
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: yes

## Issue
When running `tmux` from on Bash on Windows, `tmux` does not receive <kbd>CTRL</kbd>+<kbd>B</kbd>+<kbd>←</kbd> and <kbd>CTRL</kbd>+<kbd>B</kbd>+<kbd>→</kbd>.

It **does**, however, recieve <kbd>CTRL</kbd>+<kbd>B</kbd>+<kbd>↑</kbd> and <kbd>CTRL</kbd>+<kbd>B</kbd>+<kbd>↓</kbd>.

All of these combinations are received by `tmux` when it is running within the default Windows console.

**Comment by jwjames83** [comment]
I use CTRL+SPACE as my prefix and nothing works.

As a side note, this is also an issue on ConEmu / mintty / cmder, so it's not inherently specific to Hyper.js in the sense that other apps also miss (ignore?) these key combinations.

**Comment by jwjames83** [comment]
I use ctrl+space as my prefix on Linux, however this does not work in Windows (except for the native Windows console.) As a side note, this also does not work in other terminal emulators, such as mintty, ConEmu, and cmder.

**Comment by pachonk** [comment]
I don't think this is exclusive to windows. I am running Ubuntu (17.10 and 18.04, but this also occurs on versions lower) and I am unable to resize tmux panes left/right, but I am able to resize them up/down.
